### PR TITLE
Startup and Testpage Overlay tweaks

### PR DESCRIPTION
**Pull request by jamesagnew**
This PR:

* Tweaks the startup process to avoid overloading the database when starting up under heavy load
* Fixes a bunch of display issues in the Testpage Overlay module related to the recent upgrade to Bootstrap 5

### CODE REVIEW

File: hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/graphql/GraphQLProviderWithIntrospection.java [hunk 1]
```diff
@@ -153,37 +152,23 @@ private String generateSchema(
 			Collection<String> theResourceTypes,
 			EnumSet<GraphQLSchemaGenerator.FHIROperationType> theOperations) {
 
+		GraphQLSchemaGenerator generator = myGenerator;
```
Comment (jmarchionatto) [inline hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/graphql/GraphQLProviderWithIntrospection.java:155]: Why not simply: 
if (myGenerator == null) {
   myGenerator = new GraphQLSchemaGenerator(myContext, VersionUtil.getVersion());
}
and using myGenerator in l:160 ?

File: hapi-fhir-jpaserver-subscription/src/main/java/ca/uhn/fhir/jpa/topic/SubscriptionTopicLoader.java [hunk 2]
```diff
@@ -59,6 +61,7 @@ public SubscriptionTopicLoader() {
 	}
 
 	@Override
+	@EventListener(classes = ContextStartedEvent.class)
```
Comment (jmarchionatto) [inline hapi-fhir-jpaserver-subscription/src/main/java/ca/uhn/fhir/jpa/topic/SubscriptionTopicLoader.java:64]: Question: couldn't the overridden method be called twice, once by the super call and one by its annotation?

**Review by jmarchionatto — APPROVED** [review]
Looks good. Approved

File: hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/graphql/GraphQLProviderWithIntrospection.java (hunk 1 above)
Comment (jamesagnew) [reply hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/graphql/GraphQLProviderWithIntrospection.java:155]: The intent here was to do a [threadsafe double-check](https://medium.com/android-news/lazy-initialisation-whats-a-correct-implementation-64c4638561e) initialization, but it wasn't synchronized which it probably should be.. I'm going to just replace this with a call to Guava's memoizer.

File: hapi-fhir-jpaserver-subscription/src/main/java/ca/uhn/fhir/jpa/topic/SubscriptionTopicLoader.java (hunk 2 above)
Comment (jamesagnew) [reply hapi-fhir-jpaserver-subscription/src/main/java/ca/uhn/fhir/jpa/topic/SubscriptionTopicLoader.java:64]: Nah, have tested it and it only gets fired once.

**Pull request merged**